### PR TITLE
Creating README for list of blogs

### PR DESCRIPTION
**Pull request by dockerymick**
Signed-off-by: Mikela Jackson <mdockery@redhat.com>

### CODE REVIEW

**Comment by dockerymick** [comment]
Related issue: https://github.com/open-cluster-management/backlog/issues/12578

File: README.md [hunk 1]
```diff
@@ -31,19 +31,7 @@ You can reach the maintainers of this project at:
...
-
-* [Securing Kubernetes Clusters with Sysdig and Red Hat Advanced Cluster Management](https://www.openshift.com/blog/securing-kubernetes-clusters-with-sysdig-and-red-hat-advanced-cluster-management)
-
-* [Contributing and deploying community policies with Red Hat Advanced Cluster Management and GitOps](https://www.openshift.com/blog/contributing-and-deploying-community-policies-with-red-hat-advanced-cluster-management-and-gitops)
-
-* [Implement Policy-based Governance Using Configuration Management of Red Hat Advanced Cluster Management for Kubernetes](https://www.openshift.com/blog/implement-policy-based-governance-using-configuration-management-of-red-hat-advanced-cluster-management-for-kubernetes)
-
-* [Integrating Gatekeeper with Red Hat Advanced Cluster Management for Kubernetes](https://www.openshift.com/blog/integrating-gatekeeper-with-red-hat-advanced-cluster-management-for-kubernetes)
+Read our blogs that are in the [blogs folder](blogs/README.md)
```
Comment (dhaiducek) [inline README.md]: Maybe start this line with a `**Blogs**:` like it was before. I feel it kind of gets lost otherwise.

File: blogs/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* [Contributing and deploying community policies with Red Hat Advanced Cluster Management and GitOps](https://www.openshift.com/blog/contributing-and-deploying-community-policies-with-red-hat-advanced-cluster-management-and-gitops)
+
+## Security standards, categories, and controls
+
+* [Comply to standards using the policy based governance of Red Hat Advanced Cluster Management for Kubernetes](https://www.openshift.com/blog/comply-to-standards-using-policy-based-governance-of-red-hat-advanced-cluster-management-for-kubernetes)
+
+* [Implement Policy-based Governance Using Configuration Management of Red Hat Advanced Cluster Management for Kubernetes](https://www.openshift.com/blog/implement-policy-based-governance-using-configuration-management-of-red-hat-advanced-cluster-management-for-kubernetes)
+
+## Third-party integration blogs
```
Comment (dhaiducek) [inline blogs/README.md]: ~Probably don't need "blogs" at the end here~
Update from https://github.com/open-cluster-management/policy-collection/pull/136#issuecomment-892967904: Should be "Red Hat Supported Integrations"

**Comment by jrnp7** [comment]
Hi @dockerymick suggest following changes:
- change 'Security standards, categories, and controls' to 'Governance Overview"
- Add "Red Hat Supported  Integrations" and move OPA and Gatekeeper blogs under that - also add these Protecting Your Policies of Red Hat Advanced Cluster Management with Signature using k8s Integrity Shield https://www.openshift.com/blog/k8s-integrity-shield-tech-preview-protecting-the-integrity-of-kubernetes-resources-with-signature
and 'How to use the RHACM certificate policy controller' here

File: README.md (hunk 1 above)
Comment (dhaiducek) [inline README.md:34]: I think we should keep the `**Blogs**:` at the beginning here to set it apart from the other lines.

File: blogs/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Blogs
+
+Read our blogs for more information and best practices for Red Hat Advanced Cluster Management for Kubernetes governance capability.
+
+## Open community
```
Comment (dhaiducek) [inline blogs/README.md]: I'm wondering whether this title should be something else like "Deploying policies" or something along those lines

Comment (dockerymick) [reply blogs/README.md]: @dhaiducek What do you think about Policy deployment, to be consistent. Deploying is more active when the sections are name based on a specific noun/topic

Comment (dockerymick) [reply blogs/README.md]: Or Policy contribution and deployment?

**Comment by dockerymick** [comment]
Closing this PR. I made the changes on Gus' [PR](https://github.com/open-cluster-management/policy-collection/pull/138) bc i didn't sign-off on one of my commits.

**Pull request closed without merging**